concepts/05 UI Components/Toolbar/00 Getting Started with Toolbar/00 Getting Started with Toolbar.md

```diff
@@ -0,0 +1,22 @@
+#include tutorials-intro-installationnote
+
+The Toolbar contains items that manage the page content.
```

Comment:
The phrase "that manage the page content" limits the application of this component. I assume that items can do anything... Maybe they can add a music album to the library. This action doesn't necessarily change the current page's content. 

I don't really have a good replacement or edit for this sentence. I would simply remove it.